addons/web/static/src/js/views/kanban/kanban_renderer.js

```diff
@@ -83,6 +84,11 @@ function transformQwebTemplate(node, fields) {
 
 var KanbanRenderer = BasicRenderer.extend({
     className: 'o_kanban_view',
+    events: {
+        'click .o_kanban_mobile_tab': function(event) {
```

Comment:
handler should not be defined in events object.  it should be defined in the handlers section